stacks-common/src/types/mod.rs

```diff
@@ -72,11 +72,12 @@ pub enum StacksEpochId {
     Epoch20 = 0x02000,
     Epoch2_05 = 0x02005,
     Epoch21 = 0x0200a,
+    Epoch22 = 0x0200f,
```

Comment:
I'm not sure how these are being chosen, but since 2.1.0 (10) was `a`, should we use `14` for 2.2.0 (20), so `0x02014`?